tests/unit/amo/components/TestCollection.js

```diff
@@ -1234,8 +1234,12 @@ describe(__filename, () => {
     });
     const errorHandler = createStubErrorHandler();
     const fakeDispatch = sinon.spy(store, 'dispatch');
-    const page = 123;
+    const page = 5;
```

Comment:
😄 I noticed this one too. Do we know why it is evaluated to 5?

Reply:
yes : )

should i show the math here? basically it should be ~ numberofaddons / pageSize

Reply:
Looking at the helper functions, we don't really create 5*25 add-ons in the collection, hence my surprise. I think it is because `count = 123` in a helper function. Is that the reason?

Reply:
yeh, the count comes from 123 - from the helper function. Now that we have updated the logic to check if the page != newCollectionPage, it needs to be the correct page number here otherwise we won't get the expected results.

I could either change the count or the page to do this and went with the page...

I am not totally clear of your question maybe, so let me know if that helps

Reply:
Can we fix the `count` value to be more realistic?

Reply:
Sure